zebrad/src/application.rs

```diff
@@ -177,49 +178,11 @@ impl Application for ZebradApp {
 }
 
 impl ZebradApp {
-    fn level(&self, command: &EntryPoint<ZebradCmd>) -> String {
-        // `None` outputs zebrad usage information to stdout
-        let command_uses_stdout = match &command.command {
-            None => true,
-            Some(c) => c.uses_stdout(),
-        };
-
-        // Allow users to:
-        //  - override all other configs and defaults using the command line
-        //  - see command outputs without spurious log messages, by default
-        //  - override the config file using an environmental variable
-        if command.verbose {
-            "debug".to_string()
```

Comment:
This change removes support for the verbose logging flag `zebrad -v start`.

I'd like to keep the verbose flag, because it's really useful during development, particularly now we have removed `ZEBRAD_LOG`.

But if you'd like to remove the verbose flag, we should also remove it from the zebrad command-line options, and the flags usage message.

You can see the flags usage message by supplying an unknown flag, for example `zebrad -x`. (They really should show up in `zebrad help` with the subcommands, but that's a separate issue.)

I've looked at the abscissa docs, and I think we might need to implement our own command option parsing and help to disable this option:
https://docs.rs/abscissa_core/0.5.2/abscissa_core/command/trait.Command.html
https://docs.rs/abscissa_core/0.5.2/abscissa_core/struct.Help.html

The help is probably the more important change.

Reply:
Yes, we should get rid of this option. I didn't change it because any changes to Abscissa are currently blocked on resolving the existing in-flight changes to Abscissa. We can remove the config option from Abscissa after those changes land, but I don't think we should block these changes on that work.

Reply:
Opened https://github.com/ZcashFoundation/zebra/issues/831 for this.

Reply:
> Yes, we should get rid of this option. I didn't change it because any changes to Abscissa are currently blocked on resolving the existing in-flight changes to Abscissa. We can remove the config option from Abscissa after those changes land, but I don't think we should block these changes on that work.

I should really finish upstreaming the changes we've made to abscissa...